third_party/ngen/ngen_auto_swsb.hpp

```diff
@@ -1356,6 +1356,27 @@ inline BasicBlockList getBasicBlocks(HW hw, const Program &program)
         }
     }
 
+#ifndef NGEN_DISABLE_GETENV
+#ifndef _WIN32
+    // Check ONEAPI_PVC_SEND_WAR_WA environment variable.
+    static bool checkedEnv = false;
+    static bool haveEnv = false;
+    static bool envEnablePWW = true;
```

Comment:
Can we just initialize `envEnablePWW` with a lambda to avoid all the call once logic?

Reply:
There are two flags to initialize (haveEnv, envEnablePWW), so we'd only be saving the `checkedEnv` flag. A lambda initialization is certainly possible, but I'm not sure it would improve readability or save lines.